docs/source/early_stopping.rst

```diff
@@ -1,6 +1,17 @@
 Early stopping
 ==================
 
+Default behavior
+----------------------
+By default training will go with early stopping if there is
+`'val_loss'` in `validation_end()` return dict. Otherwise
+training will go with disabled early stopping.
```

Comment:
it might be more clear to say "Otherwise training will proceed with early stopping disabled."

Reply:
@kuynzereb could you pls rephrase it, otherwise, it is good...

Reply:
Yeah, I have already changed it. Please look at the latest changes.